packages/fx-core/package.json

```diff
@@ -65,6 +65,7 @@
     "klaw": "^3.0.0",
     "md5": "^2.3.0",
     "mime": "^2.5.2",
+    "mocked-env": "^1.3.5",
```

Comment:
This package should be in devDependencies instead of dependencies.

Reply:
thanks, updated